Firestore/core/src/firebase/firestore/model/field_path.cc

```diff
@@ -183,6 +183,22 @@ std::string FieldPath::CanonicalString() const {
   return absl::StrJoin(begin(), end(), ".", JoinEscaped());
 }
 
+/* static */
```

Comment:
FWIW, while it's not forbidden by the style guide, using `/* static */` comments is discouraged. Feel free to ignore.